test/functionalTest/cases/0501_cors/any_origin.test

```diff
@@ -40,6 +40,12 @@ orionCurl --url /v2 -X OPTIONS
 echo
 echo
 
+echo "03. Bad Verb Entry Points"
```

Comment:
This is already being tested https://github.com/telefonicaid/fiware-orion/pull/3032/files#diff-a375d95dcfcfe65f4a0395db510119dbR33, isn't it?

In that case, maybe this step is redundand in this .test and could be removed...

Reply:
Fixed in 88eeebb